src/mavedb/routers/score_sets.py

```diff
@@ -203,7 +220,11 @@ async def show_score_set(
     Fetch a single score set by URN.
     """
     save_to_logging_context({"requested_resource": urn})
-    return await fetch_score_set_by_urn(db, urn, user_data, None, False)
+    item = await fetch_score_set_by_urn(db, urn, user_data, None, False)
+    enriched_experiment = enrich_experiment_with_num_score_sets(item.experiment, user_data)
+    response_item = score_set.ScoreSet.from_orm(item).copy(update={"experiment": enriched_experiment})
+
+    return response_item
```

Comment:
Return object directly. There are other instances in this file where we should also do this, but for brevity I won't comment on all of them
```suggestion
    return score_set.ScoreSet.from_orm(item).copy(update={"experiment": enriched_experiment})
```